inc/computervirtualmachine.class.php

```diff
@@ -346,16 +346,16 @@ static function showForComputer(Computer $comp) {
          echo "<tr class='noHover'><th colspan='10'>".__('List of virtualized environments')."</th></tr>";
 
          $header = "<tr><th>".__('Name')."</th>";
-         $header .= "<th>".__('Comment')."</th>";
+         $header .= "<th>"._n('Comment', 'Comments', 1)."</th>";
          if (Plugin::haveImport()) {
             $header .= "<th>".__('Automatic inventory')."</th>";
          }
-         $header .= "<th>".__('Virtualization system')."</th>";
```

Comment:
kind of misleading classes/sentences here : 

- VirtualMachineType -> "Virtualization system"
- VirtualMachineSystem -> "Virtualization model"

The system part is the one very strange

i don't know if it's ok or not

Reply:
It's OK, I've made a replacement with sed, looking for the original string before (and I saw that part was kind of strange... I've decided I did not care :p)